main.py

```diff
@@ -147,8 +147,8 @@ def enter_Laudo_ecg(driver:webdriver.Chrome) -> Tuple[webdriver.Chrome,list]:
     date = date.strip().split('/')
 
     submitBtn = driver.find_element(By.XPATH,'//*[@id="full_search"]').click()
-    sleep(5)
-    #driver.implicitly_wait(500)
+    sleep(1)
+    driver.implicitly_wait(10)
```

Comment:
had to uncomment the implicit wait because it was messing with the time of execution